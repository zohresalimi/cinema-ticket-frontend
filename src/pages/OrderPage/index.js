import React from "react";
import TopBar from "../../components/TopBar";

function OrderPage({ children }) {
  return (
    <div>
      <TopBar />
      {children}
    </div>
  );
}

export default OrderPage;
