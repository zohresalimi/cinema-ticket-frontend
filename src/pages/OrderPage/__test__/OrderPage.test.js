import React from "react";
import { render, act } from "@testing-library/react";
import "@testing-library/jest-dom/extend-expect";

import OrderPage from "../";

jest.mock("../../../components/TopBar", () => () => (
  <div>TopBar Component</div>
));

describe("OrderPage Component Testing", () => {
  test("take snapshot", async () => {
    const { container } = await render(
      <OrderPage>
        <div>order page</div>
      </OrderPage>
    );
    expect(container.firstChild).toMatchSnapshot();
  });
});
