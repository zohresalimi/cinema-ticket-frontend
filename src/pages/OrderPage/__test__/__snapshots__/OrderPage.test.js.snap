// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`OrderPage Component Testing take snapshot 1`] = `
<div>
  <div>
    TopBar Component
  </div>
  <div>
    order page
  </div>
</div>
`;
